apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/deduplicate-entities.ai.test.ts

```diff
@@ -156,3 +156,167 @@ test(
+    name: "gpt-4-vision-preview",
+    summary:
+      "gpt-4-vision-preview is a preview model of GPT-4 with image understanding and other high-level capabilities.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "52aa0aa0-3753-4c03-8552-f86915fa5bfe",
+    name: "gpt-4-1106-vision-preview",
+    summary:
+      "gpt-4-1106-vision-preview is a preview model of GPT-4 with image understanding and advanced functionalities.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "d588f0d1-8ea1-4f18-b09e-099f1ba573eb",
+    name: "gpt-4",
+    summary:
+      "GPT-4 is a multimodal large language model known for its advanced reasoning, solving difficult problems with greater accuracy and broader general knowledge.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "02f58d32-6fa5-4055-8518-1178eaddabbb",
+    name: "gpt-4-0613",
+    summary:
+      "gpt-4-0613 is a snapshot model of GPT-4 focused on improved function calling support as of June 13, 2023.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "11a7b96a-44f0-48ee-9b88-19cd13bab94c",
+    name: "gpt-4-32k",
+    summary:
+      "gpt-4-32k is a version of GPT-4 designed for larger context windows, though not widely rolled out due to preference for GPT-4 Turbo.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "bf6aa6ae-85ea-4590-bf0e-6e9102cb69b0",
+    name: "gpt-4-32k-0613",
+    summary:
+      "gpt-4-32k-0613 is a snapshot model of GPT-4 for large context windows with better function calling support.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "1e42a1ef-d1ea-482e-97d2-37d4403591c6",
+    name: "gpt-3.5-turbo-0125",
+    summary:
+      "gpt-3.5-turbo-0125 is a version of GPT-3.5 Turbo with higher programming format accuracy and improved non-English language support.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "f20f5e21-17bb-4b27-9deb-9e04a89df33f",
+    name: "gpt-3.5-turbo-1106",
+    summary:
+      "gpt-3.5-turbo-1106 features improved instruction following and other enhancements.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+  {
+    localId: "244114fe-bd85-434b-a56b-20ebefed3210",
+    name: "gpt-3.5-turbo-instruct",
+    summary:
+      "GPT-3.5 Turbo Instruct is a large language model compatible with legacy Completions endpoints.",
+    entityTypeId:
+      "https://hash.ai/@ftse/types/entity-type/large-language-model/v/1",
+  },
+];
+
+test(
+  "Test deduplicate entities with FTSE350 companies",
```

Comment:
I think this description needs changing

Reply:
Thanks, addressed in https://github.com/hashintel/hash/pull/4505/commits/f456c9916a65fd9b416024fcc792548877ed0e3e